internal.h

```diff
@@ -1480,6 +1480,7 @@ VALUE rb_obj_methods(int argc, const VALUE *argv, VALUE obj);
 VALUE rb_obj_protected_methods(int argc, const VALUE *argv, VALUE obj);
 VALUE rb_obj_private_methods(int argc, const VALUE *argv, VALUE obj);
 VALUE rb_obj_public_methods(int argc, const VALUE *argv, VALUE obj);
+VALUE rb_obj_undefined_methods(int argc, const VALUE *argv, VALUE obj);
```

Comment:
wasn't this moved to `internal/class.h` ? 

also, what happen to this other function that was present in the original patch?

`int rb_obj_basic_to_s_p(VALUE);`